### PR TITLE
build: Show privileged test output

### PR DESCRIPTION
**Pull request by jrajahalme**
Show privileged test output in hopes this helps figuring out why they fail.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

### CODE REVIEW

**Comment by jrajahalme** [comment]
retest runtime

**Comment by jrajahalme** [comment]
retest-runtime

**Review by michi-covalent — APPROVED** [review]

**Review by joestringer — APPROVED** [review]

**Comment by pchaigno** [comment]
The test failed but the more verbose output doesn't seem to have any more indication as to what failed exactly :/

**Comment by christarazi** [comment]
Did this test run include https://github.com/cilium/cilium/pull/11620?

**Comment by jrajahalme** [comment]
retest-runtime

**Comment by jrajahalme** [comment]
@christarazi Yes, but unfortunately this does not give any better results, I thought `exit 1` might be hiding something, but alas, no. Still failing every time, now right after completing `github.com/cilium/cilium/pkg/datapath/linux/ipsec`, but before there is any output from the next iteration (`github.com/cilium/cilium/pkg/datapath/linux/probes`).

I tried running this locally by spinning up the `runtime` VM at `test` via `vagrant up runtime --provision` and then running `ginkgo --focus="Runtime.*ivileged.*" -v -- -cilium.provision=false` but locally it succeeds. Same result if I `ssh runtime` and `sudo make -C /home/vagrant/go/src/github.com/cilium/cilium/ tests-privileged`, success.

**Comment by jrajahalme** [comment]
retest-runtime (added more printing...)

**Comment by jrajahalme** [comment]
retest-runtime

**Comment by jrajahalme** [comment]
retest-runtime

**Comment by jrajahalme** [comment]
Now it fails on go test for `github.com/cilium/cilium/pkg/datapath/linux/ipsec` (I added some echoes to see bit more what is happening):
```
[2020-05-22T00:52:54.971Z]   	 Starting test in github.com/cilium/cilium/pkg/datapath/linux/ipsec...
```

**Comment by pchaigno** [comment]
It looks like the only way to reproduce is to run `retest-runtime`. [`test-focus` doesn't seem to have the same flake](https://github.com/cilium/cilium/pull/11636#issuecomment-632615323). Local testing doesn't either (I tried running the whole `Runtime*` test suite 6 times locally, all succeeding).

@nebril Is there anything specific about the `Runtime-4.9` VM that we may have missed?

**Comment by jrajahalme** [comment]
retest-runtime

**Comment by jrajahalme** [comment]
@pchaigno Trying again with `ginkgo -seed=1590090144` which runs the privileged tests within minutes from starting and failed in the past.

File: jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile [hunk 1]
```diff
@@ -132,7 +132,7 @@ pipeline {
                 TESTDIR="${GOPATH}/${PROJ_PATH}/test"
             }
             steps {
-                sh 'cd ${TESTDIR}; ginkgo --focus="$(python get-gh-comment-info.py "${ghprbCommentBody}" | sed "s/^$/Runtime.*/" | sed "s/K8s.*/NoTests/")" -v --failFast=${FAILFAST} -- -cilium.provision=false -cilium.timeout=${GINKGO_TIMEOUT}'
+                sh 'cd ${TESTDIR}; ginkgo -seed=1590090144 --focus="$(python get-gh-comment-info.py "${ghprbCommentBody}" | sed "s/^$/Runtime.*/" | sed "s/K8s.*/NoTests/")" -v --failFast=true -- -cilium.provision=false -cilium.timeout=${GINKGO_TIMEOUT}'
```
Comment (aanm) [inline jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile:135]: I assume this is unintended?

**Review by aanm — CHANGES_REQUESTED** [review]

**Comment by pchaigno** [comment]
This can probably be closed now that #11677 is merged.

**Pull request closed without merging**